test/e2e/kafka/01-connect.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: e2e-test-kafka-config
+  namespace: default
+data:
+  kaf.config: |
+    clusters:
+    - name: test
+      TLS:
+        cafile: /.kafka/certs/ca.crt
+        clientfile: /.kafka/certs/service.cert
+        clientkeyfile: /.kafka/certs/service.key
+        insecure: false
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: connect-kafka
+  namespace: default
+spec:
+  backoffLimit: 5
+  template:
+    metadata:
+      labels:
+        e2e-test: kafka
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: connect
+          image: golang:1.19
+          imagePullPolicy: IfNotPresent
+          command:
+            - bash
+          args:
+            - -c
+            - echo "Testing create topic...\n" && go install github.com/birdayz/kaf/cmd/kaf@v0.2.3 && kaf -b $KAFKA_URI --config /.kafka/config/kaf.config -c test topic create test
```

Comment:
This is not the prettiest way to do this, but surprisingly I did not find many CLIs that could properly test this

Reply:
That's test code so it's fine. I would add a new line for each new command (after `&& \`) for better readability.